src/sentry/static/sentry/app/views/releases/detail/overview/commitAuthorBreakdown.tsx

```diff
@@ -108,80 +108,52 @@ class CommitAuthorBreakdown extends AsyncComponent<Props, State> {
         <SectionHeading>{t('Commit Author Breakdown')}</SectionHeading>
         {sortedAuthorsByNumberOfCommits.map(({commitCount, author}) => (
           <AuthorLine key={author?.email}>
-            <Author>
-              <StyledUserAvatar user={author} size={20} hasTooltip />
-              <AuthorName>{userDisplayName(author || {}, false)}</AuthorName>
-            </Author>
-
-            <Stats>
-              <Commits>{tn('%s commit', '%s commits', commitCount)}</Commits>
-              <Percent>{this.getDisplayPercent(commitCount)}</Percent>
-            </Stats>
+            <UserAvatar user={author} size={20} hasTooltip />
+            <AuthorName>{userDisplayName(author || {}, false)}</AuthorName>
+            <Commits>{tn('%s commit', '%s commits', commitCount)}</Commits>
+            <Percent>{this.getDisplayPercent(commitCount)}</Percent>
           </AuthorLine>
         ))}
         {collapsedNumberOfAuthors > 0 && (
-          <StyledButton priority="link" onClick={this.onCollapseToggle}>
+          <Button priority="link" onClick={this.onCollapseToggle}>
             {tn(
               'Show %s collapsed author',
               'Show %s collapsed authors',
               collapsedNumberOfAuthors
             )}
-          </StyledButton>
+          </Button>
         )}
         {collapsedNumberOfAuthors === 0 && canExpand && (
-          <StyledButton priority="link" onClick={this.onCollapseToggle}>
-            {t('Collapse')}
-          </StyledButton>
+          <Button priority="link" onClick={this.onCollapseToggle}>
+            {t('Collapse authors')}
+          </Button>
         )}
       </Wrapper>
     );
   }
 }
 
 const AuthorLine = styled('div')`
-  display: flex;
-  justify-content: space-between;
-  margin-bottom: ${space(0.5)};
-  font-size: ${p => p.theme.fontSizeSmall};
-`;
-
-const Author = styled('div')`
-  display: flex;
-  flex: 1;
-  align-items: center;
-  overflow: hidden;
-`;
-
-const StyledUserAvatar = styled(UserAvatar)`
-  margin-right: ${space(1)};
+  display: inline-grid;
+  grid-template-columns: 30px 2fr 1fr 40px;
+  width: 100%;
+  margin-bottom: ${space(1)};
+  font-size: ${p => p.theme.fontSizeMedium};
 `;
 
 const AuthorName = styled('div')`
-  font-weight: 600;
   color: ${p => p.theme.textColor};
-  padding-right: ${space(0.5)};
   ${overflowEllipsis};
 `;
 
-const Stats = styled('div')`
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  width: 115px;
-`;
-
 const Commits = styled('div')`
-  color: ${p => p.theme.gray300};
+  color: ${p => p.theme.subText};
+  text-align: right;
 `;
 
 const Percent = styled('div')`
   min-width: 40px;
   text-align: right;
-  color: ${p => p.theme.subText};
-`;
-
-const StyledButton = styled(Button)`
-  width: 100%;
```

Comment:
Now that we are changing the centering from:
![image](https://user-images.githubusercontent.com/9060071/101739593-0c599600-3ac8-11eb-91a7-395b8575f230.png)
to:
![image](https://user-images.githubusercontent.com/9060071/101739537-f51aa880-3ac7-11eb-89fe-5b0d5728bc46.png)

I changed it also in `Other Projects for This Release` section to be consistent.